config/cloudinary.php

```diff
@@ -1,7 +1,12 @@
 <?php
 
 return [
-    "cloud_name" => "dmxebkh7h",
-    "api_key" => "726481723843648",
-    "api_secret" => "PyBAFmxpaB3eJsgoOwOghF7ih9Q"
+    // pn api keys
+    "cloud_name" => "duep6j77j",
```

Comment:
Is that right cloudinary config or you replaced it with your credentials and made commit?